tools/driver/autolink_extract_main.cpp

```diff
@@ -149,7 +150,7 @@ static bool extractLinkerFlags(const llvm::object::Binary *Bin,
         return true;
       }
     }
-    return false;
+    return Error ? true : false;
```

Comment:
Error appears to have a conversion to bool, so I don't think the select operation is needed.